common/notification/messages.ts

```diff
@@ -101,12 +101,68 @@ export async function getMessage(
     const user = await getUser(concreteNotification.userId);
     const context = getContext(concreteNotification.context as NotificationContext, user);
 
+    let headline = '';
+    try {
+        headline = template.headline(context);
+    } catch (error) {
+        logger.error(
+            `Failed to render Headline Template of Notification(${concreteNotification.notificationID}) for ConcreteNotification(${concreteNotification.id})`,
```

Comment:
🙃 It's much easier to filter messages without variables

Reply:
Bad Habits ...

Reply:
😆 but you didn't change it?

Reply:
Yeah there are many other logs as well that need to be adapted, and had to do a few other fixes ...